app/components/books-deck.js

```diff
@@ -47,6 +53,10 @@ export default Component.extend({
     filterParam(filterProperty) {
       this.sendAction('filterParam', filterProperty);
 
+    },
+    addQuantity(amount) {
+      this.set('QuantityDefault', amount);
```

Comment:
how is this adding a quantity? Please think closely about your name conventions. Is this just setting the quantity?